### PR TITLE
Design the desktop version of the portfolio

### PR DESCRIPTION
**Pull request by rica213**
Design the **header** section
Design the **headline** section
Design the **projects** section
Design the **about me** section
Design the **contact form** section
Design the **footer** with the **social media links**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -432,3 +474,482 @@ input[type="checkbox"] {
...
+
+  nav {
+    width: 50%;
+    justify-content: center;
+  }
+
+  nav ul {
+    display: flex;
+    justify-content: space-around;
+  }
```
Comment (Gopxfs) [inline style.css:505]: Please make sure to adjust the gap between your `nav` links, to follow the original design 😉:
| Figma design  | Your design |
| ------------- | ------------- |
| image  | image  |

File: style.css [hunk 2]
```diff
@@ -432,3 +474,482 @@ input[type="checkbox"] {
...
+  .project__item_one .project__name {
+    font-style: normal;
+    font-weight: 700;
+    font-size: 3.75em;
+    line-height: 80%;
+  }
+
+  .project__item_one .project__technologies {
+    margin-top: 34px;
+  }
```
Comment (Gopxfs) [inline style.css:653]: Great job adding the `ul` of technologies 😄! Please make sure to apply the corresponding `font-size` to the `li` elements 😉:
| Figma design  | Your design |
| ------------- | ------------- |
| image  | image  | 
image | image |

Kindly make sure to adjust the padding after increasing the `font-size` 😉.


Tip: 20px = 1.25em (by default)

File: style.css [hunk 3]
```diff
@@ -432,3 +474,482 @@ input[type="checkbox"] {
...
+  .geometry_three {
+    grid-column: 1/3;
+    grid-row: 5;
+    background-image:
+      url(img/desktop/image-geometry_three.svg),
+      url(img/desktop/cone_yellow.svg);
+    background-position: top left, top right;
+    background-repeat: no-repeat;
+    background-size: auto;
+  }
```
Comment (Gopxfs) [inline style.css:797]: Kindly notice that this background image isn't being displayed:
Figma design:
image

Your design:
image

Please make sure to apply all background images displayed on Figma 😉.

> Keep in mind that this project's goal is to improve your ability to follow design guidelines (imagine Microverse as a client requesting a website) 😉.

File: style.css [hunk 4]
```diff
@@ -432,3 +474,482 @@ input[type="checkbox"] {
...
+
+  .skills-container {
+    grid-column: 1/2;
+    grid-row: 1;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: 1fr 1fr;
+    row-gap: 0;
+    margin: 0;
+  }
```
Comment (Gopxfs) [inline style.css:835]: Please make sure to apply the corresponding down padding for these sections
| Figma design  | Your design |
| ------------- | ------------- |
| image  | image  | 😄:

**Review by Gopxfs — CHANGES_REQUESTED** [review]
# ♻️ STATUS: REQUIRED CHANGES ♻️

## Hi @rica213 👋,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights :rocket:
✔️ No linter errors.
✔️ Great code structure. Well done!
✔️ Your design looks flawless 😉.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 🗒️  
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Gopxfs in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — APPROVED** [review]
Hi @rica213,

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Congrats

## Good Highlights✔️

- Linter checks are passing
- Good commit messages
- Adhered to GitHub flow
- Requested change has been implemented


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**